commit dd0c72fb461a8879164588cd870702efae0c7237
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Mon Dec 20 15:15:12 2021 -0800

    Regenerate ulps on x86_64 with GCC 12
    
    Fix
    
    FAIL: math/test-float-clog10
    FAIL: math/test-float32-clog10
    
    on Intel Core i7-1165G7 with GCC 12.
    
    (cherry picked from commit de8a0897e3c084dc93676e331b610f146000a0ab)

diff --git a/sysdeps/x86_64/fpu/libm-test-ulps b/sysdeps/x86_64/fpu/libm-test-ulps
index 312575f93397c4b0..1c75f0ead46819a7 100644
--- a/sysdeps/x86_64/fpu/libm-test-ulps
+++ b/sysdeps/x86_64/fpu/libm-test-ulps
@@ -705,7 +705,7 @@ ldouble: 3
 
 Function: Real part of "clog10_towardzero":
 double: 5
-float: 5
+float: 6
 float128: 4
 ldouble: 8
 
